src/libmongoc/src/mongoc/mongoc-client.c

```diff
@@ -2151,32 +2152,44 @@ _mongoc_client_kill_cursor(mongoc_client_t *client,
    BSON_ASSERT_PARAM(collection);
    BSON_ASSERT(cursor_id);
 
-   /* don't attempt reconnect if server unavailable, and ignore errors */
-   server_stream =
-      mongoc_cluster_stream_for_server(&client->cluster, server_id, false /* reconnect_ok */, NULL, NULL, NULL);
+   bson_error_t error;
```

Comment:
```suggestion
   bson_error_t error = {};
```
Avoid uninitialized variables.